templates/docker-compose.yml.j2

```diff
@@ -94,6 +94,12 @@ services:
 {% endif %}
 {% if nim_waku_cluster_id is defined %}
       --cluster-id={{ nim_waku_cluster_id }}
+{% endif %}
+{% if nim_waku_max_msg_size is defined %}
+      --max-msg-size={{ nim_waku_max_msg_size | size }}
```

Comment:
I can't find the `size` filter in the documentation https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_filters.htm, are you sure it actually exist ?

Reply:
I doesn't apparently. Will remove